index.html

```diff
@@ -9,20 +9,44 @@
 </head>
 <body>
   <div class="container-out">
-    <h1 class="awesome">Awesome books</h1>
-    <div class="container">
-      
+    <nav class="nav">
+      <ul class="lists">
+        <li class="awesome-books"><a href="#" >Awesome Books</a></li>
+        <li class="list border-r" id="list-books" ><a href="#" id="list">List</a></li>
+        <li class="list border-r" id="addbooklink"><a href="#" id="addbook">Add Book</a></li>
+        <li class="list" id="contactlink"><a href="#"id="contact">Contact</a></li>
+      </ul>
+    </nav>
+    <div class="date"></div>
+    <h1 class="heading">All Awesome books</h1>
+    <div class="container"> 
+    </div>
+  
+    <div class="new-data">
+      <h2>Add a New Book</h2>
+      <input type="text" class="new-title" placeholder="Title">
+      <input type="text" class="new-author" placeholder="Author">
```

Comment:
[ Optional Changes ]

- [ ] What if you could clear the input fields when a book is added so the user doesn't have to manually clear them in order to add a new book?